app/controllers/application_controller.rb

```diff
@@ -25,4 +24,12 @@ def count_of_trips
   def current_user
     @user ||= User.find(session[:user_id]) if session[:user_id]
   end
+
+  def current_admin
+    current_user && current_user.role == 1
+  end
+
+  def require_admin
+    render file: '/public/404.html' unless current_admin
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.